feat: adds lw6 hackathon blog (#10781)
supabase/supabase: apps/www/_blog/2022-12-09-launch-week-6-hackathon.mdx

author: ant_wilson
image: lw6-hackathon/thumbnail.jpg
thumb: lw6-hackathon/thumbnail.jpg
tags:
  - launch-week
  - hackathon
date: '2022-12-09'
toc_depth: 3
---

The official Supabase Hackathon starts on Friday 9th December at 8am Pacific Time and ends Monday 19th December at 00:01am.

[Launch Week](https://supabase.com/launch-week) is about to kick off and we're running a hackathon in parallel.

You can win $1500 in GitHub sponsorships and an official Supabase Keyboard (extremely limited edition), along with a bunch of other prizes!

See all the [winners from all the previous hackathons](https://supabase.com/blog/tags/hackathon) for inspiration.

![Supabase Keyboard](/images/blog/lw6-hackathon/keyboard.png)

## Key Facts

- You have 10 days to build a new **Open Source** project using Supabase.
  - Starting 8:00am PT Friday 9th December 2022
- Build whatever you want - a project, app, tool, library, anything
- Enter as an individual, or as a team of up to 5 people
- Submission deadline is 00:01am early Monday morning PT 19th December 2022
- There are 5 prize categories.
  - Best overall project will win $1500 in GitHub sponsorships, and a Supabase Keyboard for each team member.

>   - Most fun/interesting, best flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!

Reviewer: ```suggestion
  - Most fun/interesting, best Flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
```

>   - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a limited edition Hackathon swag kit for each team member!

Reviewer: ```suggestion
  - Most fun/interesting, best flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
```

>   - Most fun/interesting, best flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
> - Our friends at [Deno](https://deno.land/) will be judging a special guest category: Best Edge Functions Project! They will be providing some [special swag](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon) for the winners! 

Reviewer: ```suggestion
  - Most fun/interesting, best flutter project, and best storage project will all win a limited edition Hackathon swag kit for each team member!
  - Our friends at [Deno](https://deno.land/) will be judging a special guest category: Best Edge Functions Project! They will be providing some [special swag](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon) for the winners! 
```

## Details

### Schedule

- The Hackathon begins at 8:00am PT Friday 9th December 2022
- Work on your project any time for the next 10 days
- Submission deadline (00:01am early Monday morning PT 19th December 2022)

### Prizes

There are 5 chances to win, there will be prizes for:

- Best Overall Project ($1500 in GitHub Sponsorship & Supabase Keyboards)
    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)

> - Most Visually Pleasing (Swag kit)
> - Most Fun/Interesting (Swag kit)

Reviewer: ```suggestion
- Most Fun/Interesting (Swag kit)
- Best Flutter Project (Swag kit)
```

- Best Storage Project (Swag kit)
- Best Edge Functions Project (Deno Swag kit)

> There will be a winner and a runner-up prize for each category. Every winning team member gets a prize.

Author: ```suggestion
There will be winner and runner-up prizes for each category. Every team member on winning/runner-up teams gets a supaverified swag kit.
```

  - broad usage are both ok
- FUN! 😃

### Rules

- Team size 1-5 (all team members on winning teams will receive a prize)
- You cannot be in multiple teams
- One submission per team
- All design elements, code, etc. for your project must be created **during** the event
- All entries must be Open Source (link to source code required in entry)
- Must use Supabase in some capacity
- Can be any language or framework
- You must submit before the deadline (no late entries)

### Community

The Supabase Team will be taking part in the Hackathon and you'll find us live building in our discord all week. Please join us by building in public:

- Text channel: hackathon
- Audio channel: hackathon

If you need help or advice when building, find other people to join your team, or if you just want to chill and watch people build, come and join us!

[Join our Discord](https://discord.supabase.com)

![Discord Hangout](/images/blog/hackathon/community.png)

### Launch Week

Don't forget to checkout the new feature's being announced as part of [Launch Week](/launch-week).

- [Previous Hackathon Prize Winners](https://supabase.com/blog/launch-week-5-hackathon-winners)

> - [Open Source at Supabase - Founders Fireside Chat](https://www.youtube.com/watch?v=4t_63HT3rZY)

Author: ```suggestion
- [Who We Hire at Supabase - Founders Chat](https://youtu.be/-BG9XptyCKI)
```

author: ant_wilson
image: lw6-hackathon/thumbnail.jpg
thumb: lw6-hackathon/thumbnail.jpg
tags:
  - launch-week
  - hackathon
date: '2022-12-09'
toc_depth: 3
---

# The Supabase Open Source Hackathon (Launch Week 6)

The official Supabase Hackathon starts on Friday 9th December at 8am Pacific Time and ends Monday 19th December at 00:01am.

[Launch Week](https://supabase.com/launch-week) is about to kick off and we're running a hackathon in parallel.

You can win $1500 in GitHub sponsorships and an official Supabase Keyboard (extremely limited edition), along with a bunch of other prizes!

See all the [winners from all the previous hackathons](https://supabase.com/blog/tags/hackathon) for inspiration.

![Supabase Keyboard](/images/blog/lw6-hackathon/keyboard.jpg)

## Key Facts

- You have 10 days to build a new **Open Source** project using Supabase.
  - Starting 8:00am PT Friday 9th December 2022
- Build whatever you want - a project, app, tool, library, anything
- Enter as an individual, or as a team of up to 5 people
- Submission deadline is 00:01am early Monday morning PT 19th December 2022

> - There are 6 prize categories.

Reviewer: ```suggestion
- There are 5 prize categories.
```

  - Best overall project will win $1500 in GitHub sponsorships, and a Supabase Keyboard for each team member.
  - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a limited edition Hackathon swag kit for each team member!
- Our friends at [Deno](https://deno.land/) will be judging a special guest category: Best Edge Functions Project! They will be providing some [special swag](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon) for the winners! 

![Swag kit](/images/blog/lw6-hackathon/swag.jpg)

## Details

### Schedule

- The Hackathon begins at 8:00am PT Friday 9th December 2022
- Work on your project any time for the next 10 days
- Submission deadline (00:01am early Monday morning PT 19th December 2022)

### Prizes

> There are 6 chances to win, there will be prizes for:

Reviewer: ```suggestion
There are 5 chances to win, there will be prizes for:
```

- Best Overall Project ($1500 in GitHub Sponsorship & Supabase Keyboards)
    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
- Most Visually Pleasing (Swag kit)
- Most Fun/Interesting (Swag kit)
- Best Storage Project (Swag kit)
- Best Edge Functions Project (Deno Swag kit)

There will be a winner and a runner-up prize for each category. Every winning team member gets a prize.

![Gold cap](/images/blog/lw5-hackathon/cap.png)

### Submission

You should submit your project using [this form](https://www.madewithsupabase.com/launch-week-6).

### Judges

> Best Edge Functions Project category will be judged by our friends at [Deno](https://deno.land/), and other categories will be judged by the Supabase team.

Reviewer 2: Should we mention swyx?

- Enter as an individual, or as a team of up to 5 people
- Submission deadline is 00:01am early Monday morning PT 19th December 2022
- There are 6 prize categories.
  - Best overall project will win $1500 in GitHub sponsorships, and a Supabase Keyboard for each team member.
  - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a limited edition Hackathon swag kit for each team member!
- Our friends at [Deno](https://deno.land/) will be judging a special guest category: Best Edge Functions Project! They will be providing some [special swag](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon) for the winners! 

![Swag kit](/images/blog/lw6-hackathon/swag.jpg)

## Details

### Schedule

- The Hackathon begins at 8:00am PT Friday 9th December 2022
- Work on your project any time for the next 10 days
- Submission deadline (00:01am early Monday morning PT 19th December 2022)

### Prizes

There are 6 chances to win, there will be prizes for:

- Best Overall Project ($1500 in GitHub Sponsorship & Supabase Keyboards)
    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
- Most Visually Pleasing (Swag kit)

> - Most Technically Impressive (Swag kit)

Author: ```suggestion
```

author: ant_wilson
image: lw6-hackathon/thumbnail.jpg
thumb: lw6-hackathon/thumbnail.jpg
tags:
  - launch-week
  - hackathon
date: '2022-12-09'
toc_depth: 3
---

# The Supabase Open Source Hackathon (Launch Week 6)

The official Supabase Hackathon starts on Friday 9th December at 8am Pacific Time and ends Monday 19th December at 00:01am.

[Launch Week](https://supabase.com/launch-week) is about to kick off and we're running a hackathon in parallel.

You can win $1500 in GitHub sponsorships and an official Supabase Keyboard (extremely limited edition), along with a bunch of other prizes!

See all the [winners from all the previous hackathons](https://supabase.com/blog/tags/hackathon) for inspiration.

![Supabase Keyboard](/images/blog/lw6-hackathon/keyboard.jpg)

## Key Facts

- You have 10 days to build a new **Open Source** project using Supabase.
  - Starting 8:00am PT Friday 9th December 2022
- Build whatever you want - a project, app, tool, library, anything
- Enter as an individual, or as a team of up to 5 people
- Submission deadline is 00:01am early Monday morning PT 19th December 2022

> - There are 5 prize categories.

Reviewer 3: ```suggestion
- There are 6 prize categories.
```

  - Best overall project will win $1500 in GitHub sponsorships, and a Supabase Keyboard for each team member.

>   - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a limited edition Hackathon swag kit for each team member!

Reviewer 3: ```suggestion
  - Most technically challenging, most fun, best flutter project, and most visually pleasing will all win a limited edition Hackathon swag kit for each team member!
- Our friends at [Deno](https://deno.land/) will be judging a special guest category: Best Edge Functions Project! They will be providing some [special swag](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon) for the winners! 
```

![Swag kit](/images/blog/lw6-hackathon/swag.jpg)

## Details

### Schedule

- The Hackathon begins at 8:00am PT Friday 9th December 2022
- Work on your project any time for the next 10 days
- Submission deadline (00:01am early Monday morning PT 19th December 2022)

### Prizes

> There are 5 chances to win, there will be prizes for:

Reviewer 3: ```suggestion
There are 6 chances to win, there will be prizes for:
```

- Best Overall Project ($1500 in GitHub Sponsorship & Supabase Keyboards)
    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
- Most Visually Pleasing (Swag kit)
- Most Technically Impressive (Swag kit)
- Most Fun/Interesting (Swag kit)
- Best Storage Project (Swag kit)
- Best Edge Functions Project (Deno Swag kit)

> Our friends at [Deno](https://deno.land/) have sponsored the Best Edge Functions Project category 🎉

Reviewer 3: ```suggestion
```

> You can read more about the this sponsorship [here](https://deno.com/blog/edge-functions-supabase-launch-week-6-hackathon)!

Reviewer 3: ```suggestion
```

> There will be a winner and a runner-up prize for each category. Every winning team member gets a prize.

Reviewer: @thorwebdev 
Would be great if you could take look!

![Gold cap](/images/blog/lw5-hackathon/cap.png)

### Submission

You should submit your project using [this form](https://www.madewithsupabase.com/launch-week-6).

### Judges

Best Edge Functions Project category will be judged by our friends at [Deno](https://deno.land/), and other categories will be judged by the Supabase team.

> We will be looking for:

Reviewer: @thorwebdev Here as well!
